tests/trading/test_orders.py

```diff
@@ -23,207 +29,215 @@
 WORSE_ORDER_ID = 6
 GAS_PRICE = 7
 
+ZEROED_ORDER_ID = '\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
+
 def test_walkOrderList_bids(contractsFixture):
     market = contractsFixture.binaryMarket
-    orders = contractsFixture.contracts['orders']
-    ordersFetcher = contractsFixture.contracts['ordersFetcher']
+    orders = contractsFixture.contracts['NewOrders']
+    ordersFetcher = contractsFixture.contracts['NewOrdersFetcher']
     outcomeID = 1
     order = {
         "orderID": 5,
```

Comment:
This shouldn't be `5` anymore, since `5` isn't a valid 20-byte array.  Then below we can continue doing `order["orderID"]` instead of having to reference the hard coded value.

Really all we need is a unique 20-byte array for these, we used numbers previously for simplicity but it could be a randomly generated UUID or even a 20-byte string (since Python considers strings and byte arrays to be the same).

There may be value in adding a function to `utils.py` like `longToBytes(value, length=32)` that would turn a long into a byte array padded with zeros out to `length`.  That way we could just do `longToBytes(5, 20)` here rather than having to do `\x00\x00...\x05` each time.